### PR TITLE
Fix/deploys

### PR DESCRIPTION
**Pull request by Br3nda**
Fixes the code base to get (manual) deploys to gh-page working

### CODE REVIEW

**Comment by Br3nda** [comment]
This pull request **introduces 7 alerts** when merging 2332354f70735d93601e3eda063b40f396f4e772 into 4a175a2e20ffa387c2474a08ff61d6bc9f69c4a9 - [view on LGTM.com](https://lgtm.com/projects/g/ServiceInnovationLab/serviceinnovationlab.github.io/rev/pr-42832a0064e7b4a0c59fe94f82335993b054ff79)

**new alerts:**

* 4 for Ambiguous HTML id attribute
* 2 for Useless assignment to local variable
* 1 for Missing variable declaration

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: WORKFLOW.md [hunk 1]
```diff
@@ -0,0 +1,78 @@
+# Pull Request Workflow
+
+`dev` is our main branch. Propose PRS into here
```
Comment (JacOng17) [inline WORKFLOW.md]: ADD a . at the end

File: WORKFLOW.md [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+One other dev must approve the PR before it can be merged.
+
+Approving a PR means you agree the code works as described and meets our quality levels. For most PRs this will require running the code yourself. Mention on the PR if you didn’t.
+
+If there is an associated Trello story, or github issue (waffle), provide some details and link to it. If there is no associated story, explain why briefly.
+
+## PR Feedback / Reviews
+
+As a team, we attempt to give at least one comment/feedback on each PR. This normalises asking questions. It should be unusual for a PR to have zero feedback. Feedback should be about something that couple be better. "I love this" isn't enough.
```
Comment (JacOng17) [inline WORKFLOW.md]: CHANGE couple to could.

File: WORKFLOW.md [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+Feedback is given in a form of "Have you considered..." or otherwise doesn’t assume the author didn’t already attempt your alternative approach. "Do it this other way" is not feedback.
+
+Comments can be "blocking" meaning the author wants a response to their feedback, or they can be suggestion only, leaving the decision to the PR author. Indicate this using emoji. Elephant :elephant = suggestion only.
+
+As a reviewer, you can approve a PR using the "Approve" option on github. Request changes to a PR using the "Request changes" option.
+
+## Quality
+
+Code needs to pass code linters (we use codeclimate)
```
Comment (JacOng17) [inline WORKFLOW.md]: ADD . at the end

File: WORKFLOW.md [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+
+Markup generated for webpages must pass accessibility checkers
```
Comment (JacOng17) [inline WORKFLOW.md]: ADD a . at the end

File: WORKFLOW.md [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+
+Really bad commit messages
+* fixing stuff so it works
+* please let it work this time
+* commit
+* hello
+
+## Branch Naming
+
+Branch naming in a consistent manner is good for keeping things tidy, and add context to any commit messages for commits against the branch. Branch names should be descriptive and their meanings clear. Additionally, branch names can contain `/` characters, so a good way to keep types of branches seperate is to "namespace" them - for example, any branches whose main purpose is to add a new feature to the application should have the prefix "feature". Branches for fixing bugs should be prefixed with "bugfix", while a branch for something urgent that is going to go to production asap should be prefixed with "hotfix".
```
Comment (JacOng17) [inline WORKFLOW.md]: GRAMMAR - Branch naming in a consistent manner is good for keeping things tidy and adds context...

File: WORKFLOW.md [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+
+```
+feature/automated-dishwasher-emptying
+bugfix/ui-disabled-for-anon-users
+hotfix/offline-jobs-errors
+```
+
+## Deploying
+
+Use `bin/deploy-to-gh-pages` to manually deploy. (automation coming soon). This builds from origin/dev and deploys to origin/gh-pages
```
Comment (JacOng17) [inline WORKFLOW.md]: ADD a . to the end

**Pull request merged**